build.gradle.kts

```diff
@@ -60,17 +60,20 @@ allprojects {
             }
         }
 
-        if (hasProperty("bintrayUsername") && hasProperty("bintrayApiKey")) {
-            val bintrayUsername: String by project
-            val bintrayApiKey: String by project
-            val bintrayUserOrg: String? by project
+        val bintrayUsername = "${properties["bintrayUsername"] ?: System.getenv("BINTRAY_USER")}"
```

Comment:
This is a very weird syntax!
Why use string templates?

Reply:
In fact I wasn't going for this, but:
1. `properties["bintrayUsername"]` is `Any`, so I'd need to cast it in `String?`
2. if there is no value I just have to check with `bintrayUsername.isNotEmpty()`, without checking nullability

But, if you prefer I can go with:
```kotlin
val bintrayUsername: String = properties["bintrayUsername"] as String? ?: System.getenv("BINTRAY_USER")
```
We also get rid of the nullability here.